docs/portal/developer-portal/operations/Basic_UAN_Configuration.md

```diff
@@ -64,6 +64,64 @@ CFS uses configuration layers. Configuration layers allow the sharing of Ansible
 #      branch: "{{uan.working_branch}}"
 ```
 
+### Required CFS Layers for UAN Configuration
+
+### Slingshot Host Software (playbook: shs_{{default.network_type}}_install.yml)
+
+The first CFS Layer installs the Slingshot Host Software for the Slingshot network type of the system.  The `default.network_type` is is either `mellanox` or `cassini`.
+
+### COS (playbook: cos-application.yml)
+
+The second CFS Layer runs the following roles from the `cos-config-management` VCS repository. Any configuration changes needed for these roles must be made in the `cos-config-management` `group_vars` or `host_vars` directories.
+
+The following Ansible roles are run during UAN image configuration:
+
+- #### Standard UNIX configuration
+  - cos-config-map
+  - rsyslog
+  - localtime
+  - ntp
+  - limits
+  - kdump
+- #### Allow trust of CSM generated keys for elective passwordless SSH during image customization
+
+  - trust-csm-ssh-keys
+- #### HPE Cray EX system configurations
+
+  - overlay-preload
+- #### Install COS rpms
+
+  - cos-services-install
+- #### DVS/LNET/FS
+
+  - cray_lnet_install
+  - cray_dvs_install
+  - cray_lnet_load
+  - cray_dvs_load
+
+The following Ansible roles are run during UAN post-boot configuration:
+
+- #### Standard UNIX configuration
+
+  - rsyslog
+- #### HPE Cray EX system configurations
+
+  - ca-cert
+  - overlay-preload
+- #### GPU deploy support
+
+  - cray_gpu_deploy
+
+### CMS Layer (playbook: csm_packages.yml)
+
+The third CFS Layer installs the the Cray Management System packages. These are normally not modified.
+
+### Optional UAN Layer (playbook: set_nologin.yml)
+
+The Ansible roles involved in the UAN layer which runs the `set_nologin.yml` playbook touches the `/etc/nologin` file preventing non-root users from logging into the UAN while it is being configured.
```

Comment:
1. Since this layer is optional, can we give customers guidance as to when they should (or must) use this layer and when they can skip it?
2. Just to make sure customers don't miss the dependency, we should also state that if they use this optional layer, they **must** use the `unset_nologin.yml` as well or non-root users won't be able to log into the UANs.

Reply:
Addressed comments.